src/lib.rs

```diff
@@ -407,13 +407,25 @@ impl FromPyObject<'_> for thrift_gen::jaeger::Span {
                 for reference in refs {
                     let context = reference.getattr("referenced_context")?;
                     let trace_id: u128 = context.extract_attribute("trace_id")?;
+
+                    let python_ref_type = reference.extract_attribute("type")?;
+                    let ref_type: thrift_gen::jaeger::SpanRefType =
+                        match python_ref_type {
+                            "follows_from" => thrift_gen::jaeger::SpanRefType::FollowsFrom,
+                            "child_of" => thrift_gen::jaeger::SpanRefType::ChildOf,
+                            _ => {
+                                print!(
```

Comment:
```suggestion
                                eprintln!(
```

Maybe? ie write to stderr and add a new line at the end